assets/js/src/form_editor/store/reducer.jsx

```diff
@@ -30,12 +30,18 @@ import {
   customFieldDeleteFailed,
 } from './reducers/custom_field_delete.jsx';
 import changeActiveSidebar from './reducers/change_active_sidebar';
+import {
+  disableForm,
+  enableForm,
+} from './reducers/toggle_form';
 
 const createCustomFieldStarted = createCustomFieldStartedFactory(MailPoet);
 const saveFormStarted = saveFormStartedFactory(MailPoet);
 
 export default (defaultState) => (state = defaultState, action) => {
   switch (action.type) {
+    case 'ENABLE_FORM': return enableForm(state);
```

Comment:
Just wondering, why did you decide to go with 2 reducers.action types instead one using value from action?

Reply:
No reason really. I didn't want to use a boolean attribute and so I made 2 reducer functions. And I made only a single action so that it is easier to use in the component.